test/storage-based/RedstoneDataWithClearing.test.ts

```diff
@@ -18,6 +18,11 @@ const serialized = function (x: number): number {
   return x * 10 ** 8;
 };
 
+function checkPricesDeviation(value1: number, value2: number, maxPercentageDiff: number = 5) {
```

Comment:
duplicated code, let's move it to utils.

Reply:
Done